.github/workflows/system_tests.yml

```diff
@@ -0,0 +1,68 @@
+name: System Tests
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    env:
+      RACK_ENV: test
+      RAILS_ENV: test
+      NODE_ENV: test
+      PORT: 3000
+      DISABLE_SPRING: 1
+      CAPYBARA_APP_HOST: "" # Explicitly set a blank value to override our default Docker container setup
+      NEW_RELIC_ENABLED: false
+      PAYLOAD_SECRET: amazing-payload-secret
+      AUTH_SECRET: amazing-authjs.secret # Add by executing `npx auth`. Read more: https://cli.authjs.dev
+      OIDC_ISSUER: http://wca_on_rails:3000 # Usually your local (Docker-internal!) Rails root URL
+      OIDC_CLIENT_ID: create-your-own-oauth # Create an OAuth application in the monolith with (at least!) `public`, `oidc`, `profile` and `email` scopes
+      OIDC_CLIENT_SECRET: create-your-own-oauth # See above
+      DATABASE_URI: file:./next-frontend.db
```

Comment:
All of these could be read by `dotenv`, see my other comment in the Playwright config file